pkg/clients/gcp/integration/integration_testing.go

```diff
@@ -4,7 +4,7 @@ import (
 	"flag"
 	"testing"
 
-	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega"
```

Comment:
gomega is imported bare in many test files, but this looks like the only update to fix that.  Was that intentional to not update all the instances of `. "github.com/onsi/gomega"`?  There are a lot of other bare imports too, e.g. https://github.com/crossplaneio/crossplane/blob/master/pkg/controller/aws/provider/provider_test.go#L24-L27

Reply:
Ah - at first I was confused what was happening here. The linter is configured to be a little more lenient on test files, which it considers to be files that end in `_test.go`. This file ends in `_testing.go` and thus is subject to the full barrage of stricter linters.

Reply:
Which is to say, the other instances don't cause linter warnings because they're all in test files, and the linter feels okay about bare imports in test files by default.